### DISCUSSION TITLE
Thoughts about "The non-Riemannian nature of perceptual color space" paper?

### DISCUSSION THREAD

**Discussion by blue-j** [colour-science/colour#1090] [General]
I am not knowledgeable enough to evaluate the strong claims in this paper:

[https://www.pnas.org/doi/10.1073/pnas.2119753119](https://www.pnas.org/doi/10.1073/pnas.2119753119#sec-8)

Anyone have any thoughts to share about it?

- J

**Comment by tjdcs** [comment]
The paper was very sensational. But it boils down to the idea that large color differences can't be explained my the studying the small differences between colors. 

To compare two colors for saturation or brightness, you might say that color Y is *twice* as bright as color X and this depends on a lot of factors. Such as the state of adaptation you are in, what your lighting environment is. Outdoors the illumination can reach tens of thousands of lux, which we almost never experience indoors. Yet I can still form a sort of Y is twice as bright as X kind of judgements. 

However if you study the absolute differences between those colors you will see that really Y is several times as bright as X in some circumstances. Furthermore, if you count the number of just noticeable differences (JNDs) between X and Y you might find that there are 50 JNDs (example) but if you count the number of JNDs between X and 0.5*X you might only count 30 JNDs. So the _perceptual_ scale of brightness / saturation / hue is not the same as counting the number of _discriminable_ colors. This has been known for about 70 years, at least. 

So this paper made no real contributions to advancing color science, and failed to sight about 100 years of research that has been on the topic. In the words (private communication) of a very famous color scientist, "it's just another example of physicists thinking that color can't be too hard to explain." And, in my opinion also  not acknowledging the contributions of hundreds of scientists who have dedicated their lives to the topic for the last century. 

Physicists often feel like there is no validity to color science because it starts by asking questions of real people, like "can you adjust this color Y until it's twice as bright as X" and that of course makes color scientists "soft" scientists and there's no real basis for psychological studies. Except of course.... actually there is. And a lot of numerical objective methods have been developed originally in this field.

It's not an important or useful paper if you are really interested in color science.

**Reply by blue-j** [reply]
Thank you so much for sharing your knowledge and position on this!  From my less knowledgeable position I had similar thoughts about it. It's quite cavalier.

What do you recommend to people studying color science independently?  I have years of color management experience and am not a "beginner."  But there just doesn't seem to be a solid textbook on color science that I can find!  I'm reading "Colorimetry: Understanding the CIE System"and understanding some...

-J
